bin/docpub.js

```diff
@@ -10,18 +10,19 @@
 
 const program = require('commander');
 const pkg = require('../package.json');
-const DocpubPipeline = require('../lib/docpub');
+const Docpub = require('../lib/docpub');
 
 program
     .version(pkg.version)
-    .option('-v --verbose', 'verbose output')
-    .option('-p --path <path>', 'path to documents directory (defaults to process directory)');
+    .option('--doc-path <docPath>', 'path to documents directory (defaults to process directory)')
```

Comment:
Yeah, I agree with @corycaywood , as we already have some structure, we can just add the CLI options. We can encourage the best way to use the tool with the config file should be in the same directory. Sharp cases will cover the `--config-path`.